segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/ZooKeeperServiceRunner.java

```diff
@@ -70,20 +110,24 @@ public void start() throws Exception {
             throw new IllegalStateException("Already started.");
         }
 
-        this.serverFactory.set(new NIOServerCnxnFactory());
-        val address = LOOPBACK_ADDRESS.getHostAddress() + ":" + this.zkPort;
+        if (!this.secureZK) {
```

Comment:
Is there any reason for using the NIO one or should we use the netty cnxn factory all the time?